relay-server/src/metrics_extraction/event.rs

```diff
@@ -1244,6 +1243,50 @@ mod tests {
             .any(|b| b.name == "d:spans/exclusive_time_light@millisecond"));
     }
 
+    #[test]
+    fn test_extract_span_metrics_usage() {
+        let mut event = Annotated::from_json(MOBILE_EVENT).unwrap();
+
+        // Normalize first, to make sure that all things are correct as in the real pipeline:
+        normalize_event(
+            &mut event,
+            &NormalizationConfig {
+                enrich_spans: true,
+                normalize_spans: true,
+                device_class_synthesis_config: true,
+                ..Default::default()
+            },
+        );
+
+        // Create a project config with the relevant feature flag. Sanitize to fill defaults.
+        let mut project = ProjectConfig {
+            features: [Feature::SpanMetricsExtraction].into_iter().collect(),
+            ..ProjectConfig::default()
+        };
+        project.sanitize();
+
+        let config = project.metric_extraction.ok().unwrap();
+        let metrics = extract_metrics(
+            event.value().unwrap(),
+            &config,
+            Some(&{
+                let mut o = Options::default();
+                o.span_usage_metric = true;
+                o
+            }),
+        );
+
+        let usage_metrics = metrics
+            .into_iter()
+            .filter(|b| b.name == "c:spans/usage@none")
+            .collect::<Vec<_>>();
+
+        assert_eq!(usage_metrics.len(), 7);
+        for m in usage_metrics {
+            assert!(m.tags.is_empty());
```

Comment:
Should this metric have a tag for the project?

Reply:
No, the project ID itself is not part of the bucket.